src/model/enhance-blockquotes.test.ts

```diff
@@ -10,10 +10,14 @@ describe('Enhancing blockquotes', () => {
 	});
 
 	it('adds the quoted prop when the quoted class was found', () => {
-		const input = {
+		const input: CAPIType = {
 			...example,
 			blocks: [
 				{
+					id: '123',
+					primaryDateLine: 'Wed 9 Dec 2020 06.30 GMT',
+					secondaryDateLine:
+						'Last modified on Wed 9 Dec 2020 13.40 GMT',
```

Comment:
Be good to break these out and spread them so that the intentions of the tests are kept pure